pkgs/applications/science/astronomy/stellarium/default.nix

```diff
@@ -19,28 +21,36 @@
 
 stdenv.mkDerivation rec {
   pname = "stellarium";
-  version = "1.0";
+  version = "1.1";
 
   src = fetchFromGitHub {
     owner = "Stellarium";
     repo = "stellarium";
     rev = "v${version}";
-    sha256 = "sha256-6EAykJ0yWeU1EBR5+7JjWGUVBE1DKW+W8yJOt0smkaE=";
+    sha256 = "sha256-ellfBZWOkvlRauuwug96C7P/WjQ6dXiDnT0b3KH5zRM=";
   };
 
+  # hotfix for coredump on launch: https://github.com/Stellarium/stellarium/issues/2799
+  patches = [(fetchpatch {
+    url = "https://github.com/Stellarium/stellarium/commit/8cc15658a66230ebfc2346750f443815ae210804.patch";
+    includes = [ "src/core/Dithering.hpp" ];
+    hash = "sha256-OLWfApPV33EvLpduoTFkROC44rHFKM7PSLaVFBZqSqA=";
+  })];
+
```

Comment:
Do we need this patch, when building with Qt 6? It does seem to work without the patch. Upstream also only updated the `win-qt5` assets: https://github.com/Stellarium/stellarium/releases/tag/v1.1